taworks/taform/templates/taform/ranking_status.html

```diff
@@ -30,27 +44,18 @@ <h4>Monitor the status of graduate student rankings for the upcoming term's cour
 	<th class="tg-8fvv">Instructor</th>
 	<th class="tg-8fvv">Instructor Email</th>
 	<th class="tg-8fvv">Ranking Status</th>
+	<th class="tg-8fvv">Ranking Link</th>
 	</tr>
-{% if emptyApps %}
+
 	{% for c in ranking_status %}
 <tr>
-	<td class="tg-yw4l"><a href="https://team4.uwaterloo.ca/taform/instructor/{{c.url_hash}}/">MSCI&nbsp;{{c.course_id}}</a></td>
+	<td class="tg-yw4l"><a target="_blank" href="http://www.adm.uwaterloo.ca/cgi-bin/cgiwrap/infocour/salook.pl?sess={{c.term}}&subject={{c.course_subject}}&cournum={{c.course_id}}">{{c.course_subject}} {{c.course_id}}</a></td>
 	<td>{{c.section}}</td>
-	<td>0</td>
-	<td>{{c.instructor_name}}</td>
-	<td>{{c.instructor_email}}</td>
-	<td><b>No Applicants</b></td>
-</tr>
-	{% endfor %}
-{% else %}
-	{% for c in ranking_status %}
-<tr>
-	<td class="tg-yw4l"><a href="https://team4.uwaterloo.ca/taform/instructor/{{c.course__url_hash}}/">MSCI&nbsp;{{c.course__course_id}}</a></td>
-	<td>{{c.course__section}}</td>
 	<td>{{c.count}}</td>
-	<td>{{c.course__instructor_name}}</td>
-	<td>{{c.course__instructor_email}}</td>
+	<td>{{c.instructor_name}}</td>
+	<td><a href="mailto:{{c.instructor_email}}">{{c.instructor_email}}</a></td>
 	<td>{{c.status}}</td>
+	<td><a href="https://team4.uwaterloo.ca/taform/instructor/{{c.course__url_hash}}/">Link</a></td>
```

Comment:
I'm not getting the hash for some reason: 
![screen shot 2018-03-04 at 9 13 03 pm](https://user-images.githubusercontent.com/22134649/36954605-2c94c4f2-1ff1-11e8-9bed-d192c450c3b6.png)

and the url_hash is in my course table:
![screen shot 2018-03-04 at 9 16 33 pm](https://user-images.githubusercontent.com/22134649/36954645-62ce9ab6-1ff1-11e8-84a8-edfc34b4b0a7.png)

Reply:
you have to type it in for the local http://127.0.0.1:8000/taform/instructor/E20E230BA2A445E2945F75D713/

Reply:
or the url doesnt show hash??

Reply:
nvm I understand and know what it is :) good catch

Reply:
Thanks!